app/src/main/res/layout/start_download_dialog_layout.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is a full "port" of https://github.com/mozilla-mobile/android-components/blob/main/components/feature/downloads/src/main/res/layout/mozac_downloads_prompt.xml.
This to not need another UX review but at the same time allow to avoid this rather ugly (but necessary there) [code from Fenix](https://github.com/mozilla-mobile/fenix/blob/106b692c410237642defbbb0c203d43f52ea012b/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt#L493-L501) and [this code from AC](https://github.com/mozilla-mobile/android-components/blob/d723a16e11f786f55f809e2891bb1e72aaf1c135/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/SimpleDownloadDialogFragment.kt#L97-L121).
It's simpler (and involves a lot less code) to set the values in xml and this is the difference between this layout and the one from A-C.